### PR TITLE
Ansible Service fix

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by psav** [comment]
I ordered a retest

**Comment by dajoRH** [comment]
Lint report for commit 0d17355fddcaa9ee25c6867997f1c732438cd85b:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5eebad6fa1e0d698553b6aeee6af9b61f591d7f7:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by sshveta** [comment]
The test should skip for 5.5

File: cfme/infrastructure/config_management.py [hunk 1]
```diff
@@ -294,12 +294,16 @@ def refresh_relationships(self, cancel=False):
         if not cancel:
             flash.assert_message_match(self._refresh_flash_msg)
 
+    def does_profile_exist(self):
```
Comment (psav) [inline cfme/infrastructure/config_management.py]: From now on, methods that do not navigate should be prefixed with a "_" so tha they are not intended for general use. Once this is done I am happy to merge this PR

File: cfme/tests/services/test_config_provider_servicecatalogs.py [hunk 2]
```diff
@@ -10,6 +10,10 @@
 from utils.wait import wait_for
 from cfme.services import requests
 
+pytestmark = [
+    pytest.mark.ignore_stream("5.5"),
+    pytest.mark.tier(1)]
```
Comment (dajoRH) [inline cfme/tests/services/test_config_provider_servicecatalogs.py]: I'd say tier 2

**Pull request merged**